flask/cli.py

```diff
@@ -41,10 +41,17 @@ def find_best_app(module):
 
     if len(matches) == 1:
         return matches[0]
+
+    # Finally, look for a factory function
+    factory = getattr(module, 'create_app', None)
+    if callable(factory):
```

Comment:
It could be omitted, it is a safeguard against an unusual configuration. I would consider it pythonic because it checks for an interface rather than a type. The difference between ``if factory`` and ``if callable(factory)`` seems small though.